about.html

```diff
@@ -0,0 +1,250 @@
+  <title>Porsche Cars</title>
+  <link rel="stylesheet" href="./css/styles.css">
+
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+
+  <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v6.2.0/css/all.css" />
+
+</head>
+
+<body>
+
+  <!-- HEADER SECTION STARTS -->
+  <section class="main-header">
+
+    <header class="header">
+      
+      <div class="top-navbar">
+        <div class="icons-n-text">
+          <a href="#" class="fab fa-facebook-f"></a>
+          <a href="#" class="fab fa-twitter"></a>
+          <a href="#" class="top-navbar-text">English</a>
+          <a href="#" class="top-navbar-text">My Page</a>
+          <a href="#" class="top-navbar-text">Logout</a>
+        </div>
+      </div>
+
+      <div class="second-navbar">
+
+          <a href="index.html" class="brand">
+            <h2 class="logo">Porsche</h2>
+          </a>
+
+          <nav class="navbar">
+
+            <ul class="nav-list">
+
+              <li class="nav-item">
+                <a href="about.html" class="nav-link">About</a>
+              </li>
+
+              <li class="nav-item">
+                <a href="#program" class="nav-link">Program</a>
+              </li>
+
+              <li class="nav-item">
+                <a href="#join" class="nav-link">Join</a>
+              </li>
+              
+              <li class="nav-item">
+                <a href="#sponsor" class="nav-link">Sponsor</a>
+              </li>
+
+              <li class="nav-item">
+                <a href="#news" class="nav-link">News</a>
+              </li>
+              
+              <li class="nav-item">
+                <a href="#" class="nav-link-cta">Porsche Cars</a>
+              </li>
+
+            </ul>
+
+          </nav>
+
+          <div class="icons">
+            <div class="fas fa-bars" id="menu-btn"></div>
+          </div>
+
+      </div>
+
+    </header>
+
+  </section>
+
+  <!-- HERO SECTION STARTS -->
+
+
+  <section class="hero-about">
+    <div class="hero-content-about">
+      <h5 class="hero-heading-about-subtitle">Class Is Permanent</h5>
+      <h2 class="hero-heading-about-title">We are the Porsche Club of America</h2>
+      <p class="hero-para-about">Porsche Cars North America, Inc. (PCNA), based in Atlanta, Georgia, is the exclusive
+        importer of Porsche 911, 718 Boxster and 718 Cayman sports cars
+        and the Cayenne, Macan and Panamera sports activity vehicles.
+      </p>
+
+      <p class="hero-contact-text-about">
+        Please contact us per email porsche@porsche.com or via <a href="#">1-800-767-7243</a> or
+        visit our website at <i>www.porsche.com</i> to locate a Porsche dealer near you.
+      </p>
+    </div>
+  </section>
```

Comment:
- [ ] Kindly update the text `color`, `font-size` and `font-weight` to match the design guidelines.

## Your design:
![image](https://user-images.githubusercontent.com/50721479/203854285-8ce0bcab-0268-42ac-828d-4c77bc8b4ee9.png)

## Guidelines:
![image](https://user-images.githubusercontent.com/50721479/203854314-129d5bfd-a5b1-4eb2-8739-a704f763d5e1.png)